### PR TITLE
INTLY-2003 make step 1 for each task a navigation step

### PR DESCRIPTION
**Pull request by pwright**
addresses https://github.com/integr8ly/tutorial-web-app-walkthroughs/issues/66 
and makes it standard to start a task with a navigation step

### CODE REVIEW

File: walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc [hunk 1]
```diff
@@ -285,6 +285,8 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 To use the Fuse connections, create an integration in Red Hat Fuse Online.
 
+. Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank", id="wt1a_2_3_1_fuse-url"] console.
```
Comment (pwright) [inline walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc:288]: @tonyxrmdavidson  I've reused an existing id, is that ok?

**Comment by pwright** [comment]
@briandooley  could you take a look?

**Review by briandooley — APPROVED** [review]
Looks good to me.

File: walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc (hunk 1 above)
Comment (pwright) [reply walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc:288]: chatted with @tonyxrmdavidson  - cannot use this id, and naming ids needs to be revised and documented

**Pull request merged**